ecs.yaml

```diff
@@ -1,112 +1,116 @@
 parameters:
-    exclude_files:
```

Comment:
Please configure PhpStorm to use 4 spaces for YAML and revert this formatting.

Reply:
Done.